RELEASE.md

```diff
@@ -1,3 +1,25 @@
+# Release 1.3.0
+
+## New features
+
+- Dockerfile optimisation (https://github.com/marqo-ai/marqo/pull/569/files). A pre-built Marqo base image results in reduced image layers and increased build speed.
```

Comment:
remove `/files` from the end of the PR URL because it prevents the PR from rendering properly. You can display the rich diff to see the rendered diff:

<img width="1446" alt="image" src="https://github.com/marqo-ai/marqo/assets/107458762/245e7364-cfa0-4546-81d9-27374b6f1788">